src/providers/abstract-provider.ts

```diff
@@ -583,6 +583,7 @@ export type PerformActionRequest =
     | {
           method: 'getRunningLocations';
           shard?: Shard;
+          now: boolean;
```

Comment:
What does the `now` argument do?

Reply:
now bypasses the initialization checks bc getRunningLocations needs to run during initialization.